openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -3,6 +3,16 @@
 
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
 
+// Hacky a11y overrides
+// First, these elements, while logically h3, look really odd when styled in the large h3 style
+#seq_content .wrapper--xblock .openassessment h3.response__submission__status__title{
+  font-size: 12px !important;
+}
+// This one - we were getting <header><h3/></header> elements with 2 "font-size: 1.5em;" rules
+#seq_content .wrapper--xblock .openassessment header{
+  font-size: em(16) !important;
```

Comment:
This one is different (`header` vs `h3`), let me see if removing `!important` works cleanly here.